sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -29,9 +29,46 @@ def __array__(self, dtype=None):
     raise frame_base.WontImplementError(
         'Conversion to a non-deferred a numpy array.')
 
+  astype = frame_base._elementwise_method('astype')
+
+  between = frame_base._elementwise_method('between')
+
+  @frame_base.args_to_kwargs(pd.Series)
+  @frame_base.populate_defaults(pd.Series)
+  @frame_base.maybe_inplace
+  def dropna(self, **kwargs):
+    return frame_base.DeferredFrame.wrap(
+        expressions.ComputedExpression(
+            'dropna',
+            lambda df: df.dropna(**kwargs), [self._expr],
+            preserves_partition_by=partitionings.Singleton(),
+            requires_partition_by=partitionings.Nothing()))
```

Comment:
Only tangentially related to this PR, but I realized the docstring for `ComputedExpression` is slightly incorrect: https://github.com/apache/beam/blob/a098130ae4a7d108ccacc346405ec9e03b94052e/sdks/python/apache_beam/dataframe/expressions.py#L202

Reply:
Ah, yeah, that wasn't updated when the arguments chagned. Fixed.

Reply:
Thanks!